dockerfiles/theia/src/setup.sh

```diff
@@ -2,6 +2,12 @@
 set -e
 set -u
 
+yum install -y sudo
+# Add a regular user
+useradd -u 1000 -G users,wheel,root -d ${HOME} --shell /bin/bash theia
+usermod -p "*" theia
+echo "%wheel ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
```

Comment:
all this stuff

```
yum install -y sudo
# Add a regular user
useradd -u 1000 -G users,wheel,root -d ${HOME} --shell /bin/bash theia
usermod -p "*" theia
echo "%wheel ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers

for f in "/etc/passwd" "/etc/group"; do
    chgrp -R 0 ${f}
    chmod -R g+rwX ${f};
done
# Generate passwd.template
cat /etc/passwd | sed s#root:x.*#root:x:\${USER_ID}:\${GROUP_ID}::\${HOME}:/bin/bash#g > ${HOME}/passwd.template
# Generate group.template
cat /etc/group | sed s#root:x:0:#root:x:0:0,\${USER_ID}:#g > ${HOME}/group.template
```

should go after the check of github limit to fail fast.

--> where we have `# Add yarn repo`

Reply:
Yes, thanks.